src/caselawclient/xquery/xslt_transform.xqy

```diff
@@ -6,21 +6,18 @@ declare variable $version_uri as xs:string? external;
 declare variable $img_location as xs:string? external;
 declare variable $xsl_filename as xs:string? external;
 
-let $judgment_xml := fn:doc($uri)
-let $version_xml := if ($version_uri) then fn:document($version_uri) else ()
 let $judgment_published_property := xdmp:document-get-properties($uri, xs:QName("published"))[1]
 let $is_published := $judgment_published_property/text()
 let $image_base := if ($img_location) then $img_location else ""
-let $xsl := if ($xsl_filename) then $xsl_filename else "judgment2.xsl"
-
-let $document_to_transform := if ($version_xml) then $version_xml else $judgment_xml
-let $xsl_path := fn:concat("judgments/xslts/", $xsl)
+let $document_to_transform := if ($version_uri) then fn:document($version_uri) else fn:doc($uri)
```

Comment:
the change here and removal of line
```
let $version_xml := if ($version_uri) then fn:document($version_uri) else ()
```

is just to make the code more concise right?

Reply:
yep